rotkehlchen/chain/ethereum/modules/ens/decoder.py

```diff
@@ -119,7 +119,7 @@ def __init__(  # pylint: disable=super-init-not-called
         self.graph = Graph(
             subgraph_id='5XqPmWe6gjyrJtFn9cLy237i4cWw2j9HcUJEXsP5qGtH',
             database=self.database,
-            label='ENS',
+            label=ENS_GRAPH_LABEL,
```

Comment:
why not jsut use same label as `CPT_ENS`? Does it need to be upper case? Why add another constant which is the ens constant but just upper?